docs/core/linux-prerequisites.md

```diff
@@ -103,162 +106,84 @@ On Linux, there are two installer package choices:
 
 > [!IMPORTANT]
 > Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
 
-## Install .NET Core for Red Hat Enterprise Linux (RHEL) 7
-
-To install .NET Core on RHEL 7:
-
-1. Enable the Red Hat .NET channel, available under your RHEL 7 subscription.
-    * For Red Hat Enterprise 7 Server, use:
-    
-         ```bash
-         subscription-manager repos --enable=rhel-7-server-dotnet-rpms
-         ```
-    
-    * For Red Hat Enterprise 7 Workstation, use:
-    
-        ```bash
-        subscription-manager repos --enable=rhel-7-workstation-dotnet-rpms
-         ```
-    
-    * For Red Hat Enterprise 7 HPC Compute Node, use:
-    
-        ```bash
-        subscription-manager repos --enable=rhel-7-hpc-node-dotnet-rpms
-        ```
-
-2. Install the scl tool.
+## Install .NET Core for supported Red Hat Enterprise Linux (RHEL) versions
 
-    ```bash
-    yum install scl-utils
-    ```
-    
-3. Install .NET Core
+To install .NET Core on supported RHEL versions:
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-Install .NET Core 2.0 SDK and Runtime:
-
-   ```bash
-   yum install rh-dotnet20
-   ```
-
-Enable .NET Core 2.0 SDK/Runtime for your environment:
-
-   ```bash
-   scl enable rh-dotnet20 bash
-   ```
+To ensure you have the latest installation information, follow the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/learn/get-started/linuxredhat) for supported RHEL versions.
 
 # [.NET Core 1.x](#tab/netcore1x)
 
 **.NET Core 1.1**
 
-Install .NET Core 1.1 SDK and Runtime:
+1. Remove any **previous preview** versions of .NET Core from your system.
+
+2. Install .NET Core 1.1 SDK and Runtime:
 
    ```bash
    yum install rh-dotnetcore11
    ```
 
-Enable .NET Core 1.1 SDK and Runtime for your environment:
+3. Enable .NET Core 1.1 SDK and Runtime for your environment:
 
    ```bash
    scl enable rh-dotnetcore11 bash
    ```
 
+4. Run the `dotnet --version` command to prove the installation succeeded.
+
+     ```bash
+     dotnet --version
+     ```
+     
 **.NET Core 1.0**
 
-Install .NET Core 1.0 SDK and Runtime:
+1. Remove any **previous preview** versions of .NET Core from your system.
+
+2. Install .NET Core 1.0 SDK and Runtime:
 
    ```bash
    yum install rh-dotnetcore10
    ```
 
-Enable .NET Core 1.0 SDK and Runtime for your environment:
+3. Enable .NET Core 1.0 SDK and Runtime for your environment:
 
    ```bash
    scl enable rh-dotnetcore10 bash
    ```
 
----
 4. Run the `dotnet --version` command to prove the installation succeeded.
 
      ```bash
      dotnet --version
      ```
 
 For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
+---
```

Comment:
This makes the previous line render as heading on GitHub. I think you need an empty line above it if you want `<hr>`.

Reply:
this is the syntax for the tabs, but I agree with you that it probably needs an empty line above it @svick. Right now the layout is broken here.